### ISSUE TITLE
Fix Datasync jobs for B2, Schedule H and Form 496

### ISSUE THREAD

**Issue by lla2105** [daguar/netfile-etl#4]
I have set up 16 jobs on Datasync to update 16 datasets on Socrata. Fortunately 13 of the Datasync jobs work very time I run them.However the following 3 Socrata Datasync jobs do NOT work at all. 

Form-460-Schedule-B2-Loan
Form-460-Schedule-H-Loans
Form-496-Late-Independent--Expenditures

**Comment by daguar** [comment]
Hey Lauren! Looking at some data from a few weeks ago, those appear to be empty. That's probably why Socrata doesn't want to accept the upload.

How do you want to approach this?

I'd suggest just letting them fail; that way, when data _does_ exist, the jobs will run and put the data up there.

**Comment by lla2105** [comment]
Yeah that might be the reason but I think datasync has uploaded some datsets just fine that do not currently have any data in them. I’ll look into it more though – thanks for the feedback!

**Comment by daguar** [comment]
Ah! Well in that case that'd be my first step -- check if any other data
sets are empty. If they're not, then that's almost assuredly the reason.

On Mon, Apr 7, 2014 at 3:59 PM, lla2105 notifications@github.com wrote:

> Yeah that might be the reason but I think datasync has uploaded some
> datsets just fine that do not currently have any data in them. I'll look
> into it more though - thanks for the feedback!
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/daguar/netfile-etl/issues/4#issuecomment-39794156
> .

## 

Dave Guarino
Consultant, Health Vertical
(2013 Fellowship alumnus)
Code for America http://www.codeforamerica.org/
dave@codeforamerica.org
LinkedIn http://www.linkedin.com/in/daveguarino/ |
GitHubhttps://github.com/daguar/
